src/components/form/Dropdown.tsx

```diff
@@ -70,19 +90,27 @@ export const Dropdown = ({
     <div className="district-project-search-form__filter">
       <Combobox
         clearButtonAriaLabel={clearButtonAriaLabel}
+        disabled={loading}
         label={label}
         icon={icon}
         // @ts-ignore
         options={options}
         onChange={(values: OptionType[]) => {
-         let uniqueValues = values.filter((val, index, array) => array.findIndex(t => t.value === val.value) === index);
-          setValue(uniqueValues);
+          let uniqueValues = values.map(({label, ...values}) => values).filter((val, index, array) => array.findIndex(t => t.value === val.value) === index);
```

Comment:
Is this an issue with data (should we be using IDs here instead). Are there duplicate values? Might be nothing, just seemed a bit odd